lib/controllers/backend/spree/admin/admin_controller_decorator.rb

```diff
@@ -1,14 +1,20 @@
 # frozen_string_literal: true
 
-Spree::Admin::BaseController.class_eval do
```

Comment:
file should be named `base_controller_decorato.rb` right?

Reply:
I actually have seen this pattern in several places, but I agree with you. Will make that change.